tools/nym-cli/envs/mainnet.env

```diff
@@ -0,0 +1,20 @@
+CONFIGURED=true
```

Comment:
I think we should keep all of those `.env` files in a separate directory since they are used in other binaries besides `nym-cli`

Reply:
Yeah, could move them to the `NetworkDefaults` - they were really there as examples. Maintenance of them is an issue too.